files/task_helper.rb

```diff
@@ -54,11 +54,11 @@ def self.run
       STDOUT.print result.to_s
     end
   rescue TaskHelper::Error => e
-    STDOUT.print(e.to_h.to_json)
+    STDOUT.print({ _error: e.to_h }.to_json)
     exit 1
   rescue StandardError => e
     error = TaskHelper::Error.new(e.message, e.class.to_s, e.backtrace)
-    STDOUT.print(error.to_h.to_json)
+    STDOUT.print({ _error: error.to_h }.to_json)
```

Comment:
Would it be worth adding a failure status? That way with human outputter there is not an empty hash under the error message.
```suggestion
STDOUT.print({status: 'failure',  _error: error.to_h }.to_json)
```
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt$ bolt task run test::ruby_task_helper name=hi -t localhost
Started on localhost...
Failed on localhost:
  ooops task failed
  {
    "status": "failure"
  }
Failed on 1 node: localhost
Ran on 1 node in 0.19 sec
```

Reply:
@lucywyman @nicklewis I think we agree on the version. What do ya'll think about the status key?

Reply:
I don't like the status key because we already define success/failure as part of the task spec. Also, there is no status key in the success case, which mean it's not something you can actually rely on.

If the problem is a superfluous empty hash being printed after the error message, we could fix that in bolt rather than changing the result here.